api/v1/path.go

```diff
@@ -147,3 +147,15 @@ const (
 	// Response: SearchOpenCorporatesResponse
 	PathForMartiniSearchCorps = "/v1/ms/search/opencorporates"
 )
+
+// FCC service API
+const (
```

Comment:
while you  were working on this we already introduced `martini.go`

let's move `/fcc` under `/v1/ms`:
something like:
`/v1/ms/fcc_frn`
`/v1/ms/fcc_search_detail`